static/app/views/issueList/overview.tsx

```diff
@@ -333,6 +338,8 @@ class IssueListOverview extends Component<Props, State> {
   }
 
   getBetterPriorityParams(): BetterPriorityEndpointParams {
+    const user = ConfigStore.get('user');
+    const variant = user.experiments?.PrioritySortExperiment || 'baseline';
```

Comment:
@ceorourke shouldn't we check the early access flag here since teh user experiment won't check it?

Reply:
@ceorourke you could actually put the feature flag check here too